flow/flow.go

```diff
@@ -108,14 +108,17 @@ const (
 	DNSLayer ExtraLayers = 2
 	// DHCPv4Layer extra layer
 	DHCPv4Layer ExtraLayers = 4
+	// DNSLayerAll extra layer, intercept every packet
+	DNSLayerAll ExtraLayers = 8
 	// ALLLayer all extra layers
-	ALLLayer ExtraLayers = 255
+	ALLLayer ExtraLayers = 247
```

Comment:
any reason to change this ?

Reply:
Just did this so that if one requests all layers they won't have to, by default, capture all the packets. But since we are only keeping one option I will modify this again

Reply:
done